### ISSUE TITLE
Related content links without primary navigation broken at mobile widths

### ISSUE THREAD

**Issue by Limess** [Financial-Times/o-header-services#88]
Type: Bug

## What
Version `3.x.x` has broken layout for related content links at `M` breakpoints and below when no primary navigation exists. Observed on `3.2.6`.

## Details

Relevant `o-header-services` example: https://www.ft.com/__origami/service/build/v2/demos/o-header-services@3.2.5/title-section?brand=internal. There's no equivalent demo of this working as the `2.x.x` demos do not include related content links without a primary nav.

### Previous behaviour

Example page using `v2.x.x` with no primary navigation: https://heimdall.in.ft.com/

Full width:

![screenshot_2019-02-28 heimdall team list 6](https://user-images.githubusercontent.com/3199181/53574930-7e1d5580-3b68-11e9-8d2e-92389d7ec0f2.png)

Mobile: 
![screenshot_2019-02-28 heimdall team list 5](https://user-images.githubusercontent.com/3199181/53574928-7e1d5580-3b68-11e9-8d8b-4e4c96fffc3c.png)

The related content links are hidden in the burger menu.

### New behaviour

On `3.2.6`

![screenshot_2019-02-28 heimdall team list 7](https://user-images.githubusercontent.com/3199181/53575121-d5bbc100-3b68-11e9-876d-0388122d8b07.png)

The related content links are scrunched up with broken styling and the burger icon is non-interactive as the javascript does not fire due to this check for primary navigation: https://github.com/Financial-Times/o-header-services/blob/a1822d6974f20e454fb805f82a7b2b3098e7db6a/src/js/drawer.js#L16

The source for this component is available here:

[header.jsx](https://github.com/Financial-Times/heimdall-ui/blob/822240d3c92c246c853845067f272e2fe8d02c2b/src/components/header.jsx)
[_header.scss](https://github.com/Financial-Times/heimdall-ui/blob/822240d3c92c246c853845067f272e2fe8d02c2b/src/styles/_header.scss)

## Additional information

I initially raised this on Slack https://financialtimes.slack.com/archives/C02FU5ARJ/p1551360410011600

**Comment by Limess** [comment]
This seems to be fixed by adding the markup for the primary nav with no nav items:

```html
<div class="o-header-services__primary-nav">
	<div class="o-header-services__primary-nav-list"></div>
</div>
```

**Comment by Limess** [comment]
Example of the flash:

![loading](https://user-images.githubusercontent.com/3199181/53577117-000f7d80-3b6d-11e9-8aa9-74dff01878da.gif)

This is running locally (dev build with webpack-dev-server) so with a throttled connection so is a bit jumpier than it would be in prod...

**Comment by notlee** [comment]
Sorry this has been sat around a while @Limess. This should improved by: https://github.com/Financial-Times/o-header-services/pull/97 and https://github.com/Financial-Times/o-header-services/pull/98

**Comment by notlee** [comment]
This should be fixed in v3.2.13 :)

**Comment by Limess** [comment]
Smashing, thanks!
